src/rtc/mosaic_geobursts.py

```diff
@@ -186,73 +273,184 @@ def compute_weighted_mosaic_array(list_rtc_images, list_nlooks,
 
         dim_mosaic = (geogrid_in.length, geogrid_in.width)
 
-        xmax_mosaic = xmin_mosaic + posting_x*dim_mosaic[1]
-        ymin_mosaic = ymax_mosaic + posting_y*dim_mosaic[0]
-        dim_mosaic = (geogrid_in.length, geogrid_in.width)
+        xmax_mosaic = xmin_mosaic + posting_x * dim_mosaic[1]
+        ymin_mosaic = ymax_mosaic + posting_y * dim_mosaic[0]
 
         srs_mosaic = osr.SpatialReference()
         srs_mosaic.ImportFromEPSG(geogrid_in.epsg)
         wkt_projection = srs_mosaic.ExportToWkt()
 
     if verbose:
-        print(f'mosaic dimension: {dim_mosaic}, #bands: {num_bands}')
+        print(f'    mosaic dimension: {dim_mosaic}, number of bands: {num_bands}')
```

Comment:
What do you think about printing out the projection of the mosaic when in verbose mode?